packages/db/src/migrations/20250915152638_create_granule_duplicates_table_and_index.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  await knex.schema.createTable('granule_duplicates', (table) => {
+    table
+      .increments('cumulus_id')
+      .primary();
+    table
+      .integer('granule_cumulus_id')
+      .references('cumulus_id')
+      .inTable('granules')
+      .notNullable();
+    table
+      .specificType('status', 'char(1)')
+      .comment('Granule active status')
+      .notNullable();
+    table
+      .string('group_id')
+      .comment('Granule duplicate group id')
+      .notNullable();
```

Comment:
So each granule _involved in a duplicate scenario_ would have an entry in this table? e.g. if there are two granules with different granule_cumulus_ids, both would have records here with the same group_id? And `status` defines which of them is active?